bundles/org.openhab.binding.asuswrt/src/main/java/org/openhab/binding/asuswrt/internal/things/AsuswrtClient.java

```diff
@@ -0,0 +1,245 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.asuswrt.internal.things;
+
+import static org.openhab.binding.asuswrt.internal.constants.AsuswrtBindingConstants.*;
+import static org.openhab.binding.asuswrt.internal.helpers.AsuswrtUtils.*;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.binding.asuswrt.internal.helpers.AsuswrtUtils;
+import org.openhab.binding.asuswrt.internal.structures.AsuswrtClientInfo;
+import org.openhab.binding.asuswrt.internal.structures.AsuswrtTraffic;
+import org.openhab.core.config.core.Configuration;
+import org.openhab.core.library.unit.Units;
+import org.openhab.core.thing.ChannelUID;
+import org.openhab.core.thing.Thing;
+import org.openhab.core.thing.ThingStatus;
+import org.openhab.core.thing.ThingTypeUID;
+import org.openhab.core.thing.binding.BaseThingHandler;
+import org.openhab.core.types.Command;
+import org.openhab.core.types.RefreshType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * CLASS HANDLING ROUTER CLIENTS
+ *
+ * @author Christian Wild - Initial contribution
+ */
+@NonNullByDefault
+public class AsuswrtClient extends BaseThingHandler {
+    private final Logger logger = LoggerFactory.getLogger(AsuswrtClient.class);
+    private final AsuswrtRouter router;
+    private Map<String, Object> oldStates = new HashMap<>();
+    protected final String uid;
+
+    /**
+     * Constructor
+     *
+     * @param thing Thing object representing client
+     * @param router Router (Bridge) Thing
+     */
+    public AsuswrtClient(Thing thing, AsuswrtRouter router) {
+        super(thing);
+        this.router = router;
+        this.uid = getThing().getUID().getAsString();
+    }
+
+    /***********************************
+     *
+     * INIT AND SETTINGS
+     *
+     ************************************/
+
+    /**
+     * INITIALIZE DEVICE
+     */
+    @Override
+    public void initialize() {
+        logger.trace("({}) Initializing thing ", uid);
+        router.queryDeviceData(false);
+        refreshData();
+        updateStatus(ThingStatus.ONLINE);
+    }
+
+    /***********************************
+     *
+     * COMMAND AND EVENTS
+     *
+     ************************************/
+    /**
+     * handle Commands
+     */
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        if (command instanceof RefreshType) {
+            refreshData();
+        }
+    }
+
+    /**
+     * update clientpropedrties
+     * 
+     * @param clientInfo
+     */
```

Comment:
The code contains a lot of redundant JavaDocs with typos that only add clutter and add nothing useful.
It would be better to remove them all. Please also do this in the other files.

```suggestion
```

Reply:
Why is adding docs "clutter"? I think it is easyer and clearer to read code if there are JavaDocs.
Ok maybe i should be better to explain more in the docs but remove them all?

Reply:
If the docs document something useful I'd agree. But they now just copy/paste what's already in the method declaration. I.e. they only add more vertical scrolling. Most of the time you do not need any JavaDocs if you write clean code and the functionality should be clear from the method name and parameters. It will also cause the good docs that are written to stand out more and cause devs to actualy spend time reading them because they document something important.

Reply:
So.... because i always write clean code i can remove any comments. :joy:  :joy:
Ok i will remove the `@param`, etc or add new comments if it will help

Reply:
At the end of the review it will be clean code. :wink: